.github/workflows/create-new-hack.yml

```diff
@@ -2,3 +2,35 @@ name: Create new hack
 
 on:
   workflow_dispatch:
+    inputs:
+      hackName:
+        description: The name of the hack you want to create
+        required: true
+        default: YetAnotherHack
+      numberOfChallenges:
+        description: The number of challenges you want
+        required: true
+        default: 3
+jobs:
+  createNewWhatTheHackTemplate:
+    runs-on: ubuntu-latest
+    env:
+      BRANCH_NAME: xxx-${{ github.event.inputs.hackName }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Make script executable
+        run: |
+          chmod +x ${{ github.workspace }}/.github/workflows/create-wth-template.sh
```

Comment:
Hey Jordan, we can checkin the file with the +x flag using the following:

git update-index --chmod=+x path/to/file

This would allow you to skip this step every run.

Reply:
good point, I forgot about that option, fixed